charts/quantum-serverless/charts/gateway/templates/deployment.yaml

```diff
@@ -65,6 +65,8 @@ spec:
           env:
             - name: DEBUG
               value: {{ .Values.application.debug | quote }}
+            - name: DJANGO_SECRET_KEY
```

Comment:
I think we can consider that this value will be usually taken from a secret store as the [`DATABASE_PASSWORD`](https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/charts/quantum-serverless/charts/gateway/templates/deployment.yaml#L118). I would change to something like:

```
            - name: DJANGO_SECRET_KEY
              valueFrom:
                secretKeyRef:
                  name: django
                  key: secret_key
```

what do you think?

Reply:
Could not agree more! Good catch!